app/browser/api/ledger.js

```diff
@@ -226,20 +230,83 @@ const checkSeed = (state) => {
   return state
 }
 
-const getPublisherTimestamp = (updateList) => {
+const getPublisherInfo = () => {
   if (!client) {
     return
   }
 
-  client.publisherTimestamp((err, result) => {
+  client.fetchPublisherInfo((err, result) => {
     if (err) {
-      console.error('Error while retrieving publisher timestamp', err.toString())
+      console.error('Error while retrieving verified publishers', err.toString())
       return
     }
-    appActions.onPublisherTimestamp(result.timestamp, updateList)
+    appActions.onPublishersInfoReceived(result)
   })
 }
 
+const checkPublisherInfoUpdate = (state) => {
+  const verifiedPTimestamp = updateState.getUpdateProp(state, 'verifiedPublishersTimestamp') || null
+
+  if (
+    verifiedPTimestamp == null ||
+    ((new Date().getTime() - verifiedPTimestamp) >= publisherInfoUpdateInterval)
+  ) {
+    if (publisherInfoTimeoutId) {
+      clearTimeout(publisherInfoTimeoutId)
+    }
+
+    // Startup and boot delay
+    const delay = (!bootP || !client) ? (ledgerUtil.milliseconds.second * 15) : 0
+
+    publisherInfoTimeoutId = setTimeout(() => {
+      module.exports.getPublisherInfo()
```

Comment:
Is it possible to hose state here? (Does it need an appAction)

Reply:
Disregard. Jumped the gun on that.

Reply:
👍